### PR TITLE
Disable Android hardware back button on main screen

### PR DESCRIPTION
**Pull request by mikeverf**
> Disable Android hardware back button on main screen
> and enable it on personalSchedule

This PR closes ticket #77

### CODE REVIEW

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-77-fix_android_hardware_button

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-77-fix_android_hardware_button

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

File: screens/home/HomeContainer.js [hunk 1]
```diff
@@ -28,6 +28,10 @@ type TTrackSession = {
 type TTrackSessions = Array<TTrackSession>;
 
 export default class HomeContainer extends Component<TProps> {
+  componentWillMount() {
+    BackHandler.addEventListener('hardwareBackPress', () => BackHandler.exitApp());
```
Comment (thibmaek) [inline screens/home/HomeContainer.js]: Would check here if the page you're _going to navigate to_ is Login _and_ there is a logged in user in state/storage, only then activate the backhandler

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-77-fix_android_hardware_button

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-77-fix_android_hardware_button

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-77-fix_android_hardware_button

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

File: screens/screenConfig.js [hunk 2]
```diff
@@ -9,37 +9,45 @@ export const MAIN = 'Main';
 
 type TRouteName = HOME | LOGIN;
 
-const createMainNavigator = (initialRouteName: TRouteName): StackNavigatorConfig =>
+const createMainNavigator = (): StackNavigatorConfig =>
```
Comment (thibmaek) [inline screens/screenConfig.js]: Param is deleted here, but still passed in the file above this. Either this needs the param back or it shouldn't be passed to createRootSwitchNavigator

**Comment by devopsITP** [comment]
:shipit: This branch has been deployed to:
  https://expo.io/@itpocket/white-label-event-mobile-app-feature-77-fix_android_hardware_button

  Download the [Expo](https://expo.io/) app and scan this QR code to get started!

  QR Code

**Pull request merged**